docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -774,9 +754,36 @@ Users can request a new, valid password using the "Forgot password" feature.
 
 Strict types have been added to Public API methods.
 
-## ezplatform-matrix-fieldtype
+### Zeta Components (eZc) Database handler
+
+The deprecated Zeta Components (eZc) Database handler has been dropped.
+All classes and interfaces from `eZ\Publish\Core\Persistence\Database` and `eZ\Publish\Core\Persistence\Doctrine` namespaces have been removed.
+
+`ezpublish.connection` has been removed. Use `ezpublish.persistence.connection` instead.
+
+The signature of the `\eZ\Publish\Core\Persistence\Legacy\URL\Query\CriterionHandler::handle` contract
+now accepts `\Doctrine\DBAL\Query\QueryBuilder` instead of `\eZ\Publish\Core\Persistence\Database\SelectQuery` and has the following form:
+
+``` php
+use \Doctrine\DBAL\Query\QueryBuilder;
+use \eZ\Publish\Core\Persistence\Legacy\URL\Query\CriteriaConverter;
+use \eZ\Publish\API\Repository\Values\URL\Query\Criterion;
+public function handle(CriteriaConverter $converter, QueryBuilder $query, Criterion $criterion);
```

Comment:
TBH on the output it doesn't look good without a blank line separating imports from the prototype.